.travis.yml

```diff
@@ -0,0 +1,40 @@
+dist: trusty
+
+language: go
+
+go:
+  - 1.7
+  - tip
+
+go_import_path: github.com/docker/containerd
+
+addons:
+  apt:
+    packages:
+      - btrfs-tools
+
+install:
+  - wget https://github.com/google/protobuf/releases/download/v3.1.0/protoc-3.1.0-linux-x86_64.zip -O /tmp/protoc-3.1.0-linux-x86_64.zip
+  - unzip -o -d /tmp/protobuf /tmp/protoc-3.1.0-linux-x86_64.zip
+  - export PATH=$PATH:/tmp/protobuf/bin/
+  - go get golang.org/x/net/context
+  - go get github.com/gogo/protobuf/gogoproto
+  - go get github.com/gogo/protobuf/proto
+  - go get github.com/nats-io/go-nats
+  - go get github.com/opencontainers/runtime-spec/specs-go
+  - go get github.com/sirupsen/logrus
+  - go get github.com/tonistiigi/fifo
+  - go get github.com/urfave/cli
+  - go get google.golang.org/grpc
+  - go get google.golang.org/grpc/grpclog
+  - go get github.com/crosbymichael/go-runc
+  - go get github.com/docker/go-metrics
+  - go get github.com/nats-io/nats-streaming-server/server
+  - go get github.com/opencontainers/runc/libcontainer
+  - go get github.com/docker/distribution/digest
+  - go get github.com/stevvooe/go-btrfs
+  - go get github.com/nightlyone/lockfile
+  - go get github.com/stretchr/testify
```

Comment:
Please don't list all this. `go get` does this automatically with `go get -d ./...`.